src/firefly/js/templates/lightcurve/LcPeriodogram.jsx

```diff
@@ -268,7 +272,7 @@ class PeriodogramOptionsBox extends Component {
                         </button>
                         <br/>
                         <div style={{marginTop: 10}}>
-                            {'Leave the fields blank to use default values.'}  <HelpIcon helpId={'periodogram.options'}/>
+                            {'Leave the fields blank to use default values.'}  <HelpIcon helpId={'findpTSV.pgramresults'}/>
```

Comment:
this icon position could be adjusted to the lower right corner of the option panel, or line up with the "Periodogram Calculation" button